tests/phpunit/test-cache.php

```diff
@@ -81,7 +111,6 @@ public function test_redis_reload_connection_closed() {
 		$this->assertFalse( $this->cache->redis->IsConnected() );
 		// Reload occurs with set()
 		$this->cache->set( 'foo', 'banana' );
-		$this->assertEquals( 'WP Redis: Connection closed', $this->cache->last_triggered_error );
```

Comment:
@danielbachhuber I removed this because Predis will automatically reconnect internally whenever it's disconnected and an action is trying to be performed, therefore not throwing any "Connected Closed" errors. As the next 3 tests essentially verify what this test is verifying, I felt it was safe to remove this particular one. Thoughts?